sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

```diff
@@ -131,7 +133,40 @@ action `unmarkFalsePositives`
 include::sechub_client_falsepositve_list_example_unmark.json[]
 ----
 
-==== Options
+===== interactiveMarkFalsePositives
+Instead of creating a json file to mark false positives, you can also do this interactively:
+Go to the project directory containing `sechub.json` and a report file from a recent scan and run `sechub interactiveMarkFalsePositives`.
+
+You can select the report file using the `-file` option of the client. Otherwise the latest report file in the current directory is automatically selected.
+
+For each finding in the reports file, you are prompted: `"Add this as false positive?"`
+Your choices are:
+
+- `y` Yes - Mark as false positive. You are prompted for an optional comment line which can help later to understand why this has been marked.
+- `n` No - Do nothing and go to next item.
+- `s` Skip following findings - End the asking here and add the marked items to the false positives list on the sechub server.
```

Comment:
This option is generally hard to understand. I would have expected, that only one finding will be skipped. This option skips all remaining findings. Which is helpful. However, the current name is confusing.

Reply:
`s` was the proposal.
If you think, that is ugly, please open an issue for this so it can be discussed in the team.